vllm/config/scheduler.py

```diff
@@ -93,6 +93,9 @@ class SchedulerConfig:
     is_multimodal_model: bool = False
     """True if the model is multimodal."""
 
+    is_encoder_decoder: bool = False
+    """True if the model is an encoder-decoder model."""
+
```

Comment:
If this already exists in `ModelConfig`, why duplicate it here?

Reply:
The `InitVar` solution works.

However, in other cases like this (where two sibling configs interact) I've tended to perform those interractions in the parent's `__post_init__`, `VllmConfig` in this case. Would that work in this case?

Reply:
That's where I had it before this change, but we end up with a confusing log message about features being enabled coming from the SchedulerConfig's __post_init__ before VllmConfig's __post_init__ fixed it and disabled them.